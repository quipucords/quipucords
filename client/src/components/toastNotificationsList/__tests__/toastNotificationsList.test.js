import React from 'react';
import configureMockStore from 'redux-mock-store';
import { shallow } from 'enzyme';
import ToastNotificationsList from '../toastNotificationsList';

describe('ToastNotificationsList Component', () => {
  const generateEmptyStore = () => configureMockStore()({ toastNotifications: {} });

  it('should shallow render a basic component', () => {
    const store = generateEmptyStore();
    const props = { show: true };
    const wrapper = shallow(<ToastNotificationsList {...props} />, { context: { store } });

    expect(wrapper.dive()).toMatchSnapshot();
  });
});
