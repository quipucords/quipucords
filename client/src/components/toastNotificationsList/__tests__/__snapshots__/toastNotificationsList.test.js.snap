// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`ToastNotificationsList Component should shallow render a basic component 1`] = `
<ToastNotificationList
  className=""
  onMouseEnter={[Function]}
  onMouseLeave={[Function]}
  onMouseOver={[Function]}
/>
`;
